apps/web/src/state/types.ts

```diff
@@ -14,9 +14,9 @@ import { Address, parseEther } from 'viem'
 import { NftToken } from './nftMarket/types'
 
 export enum GAS_PRICE {
-  default = '1',
-  fast = '4',
-  instant = '5',
+  default = '5',
```

Comment:
any context on this update?

Reply:
@thechefpenguin this one update from the commits 2 weeks ago from you, could you help to take a look?

Reply:
@chefjackson This is coming from the values in design file. It is only for BSC.